### PR TITLE
added more comments inside of groupView.ts and groupView.html, also a…

### PR DESCRIPTION
**Pull request by nbhamilton94**
…dded footer for chat button and styled according to the spec.

### CODE REVIEW

File: src/pages/groupView/groupView.html [hunk 1]
```diff
@@ -1,19 +1,11 @@
...
-  <ion-navbar >
+  <ion-navbar>
 	<ion-buttons>
 		<button class="menu-icon" ion-button icon-only>
-			<ion-icon name="menu"></ion-icon>
+			<ion-icon name="menu" color="primary"></ion-icon>
 		</button>
 	</ion-buttons>
-    
-
```
Comment (khen27) [inline src/pages/groupView/groupView.html]: nit: you have white space here - what's the coding standard for this? If you aren't sure, follow up with @evanBRAQS

File: src/pages/groupView/groupView.html [hunk 2]
```diff
@@ -1,19 +1,11 @@
...
-
-    <ion-title align="center">
-    Groups
-
```
Comment (khen27) [inline src/pages/groupView/groupView.html]: nit: recycling bits - go through and clean up the white spaces after confirming what the standard is.

File: src/pages/groupView/groupView.html [hunk 3]
```diff
@@ -35,9 +27,14 @@
 		<hr>
 	</ion-list>
 
-	<button id="create-group-button" color="primary" ion-fab>
```
Comment (khen27) [inline src/pages/groupView/groupView.html]: I noticed you commented the other html tags above, but not this one, what's the standard here?

File: src/pages/groupView/groupView.html [hunk 4]
```diff
@@ -35,9 +27,14 @@
...
-		<ion-icon name="md-chatbubbles" ></ion-icon>
```
Comment (khen27) [inline src/pages/groupView/groupView.html]: recycling bits - what's the standard on commenting?

File: src/pages/groupView/groupView.scss [hunk 5]
```diff
@@ -5,15 +5,17 @@
     right: 1px;
     overflow: visible!important;
     float: right;
+    font-size: 1.2em;
 }
 
-
```
Comment (khen27) [inline src/pages/groupView/groupView.scss]: I would follow up to ensure we don't need comments for scss files.

File: src/pages/groupView/groupView.ts [hunk 6]
```diff
@@ -8,8 +8,9 @@ import { GroupChatView } from '../groupChatView/groupChatView';
 })
 
 export class GroupViewPage {
-
+	//I'm defining groups, which would be an array of objects in this example.
```
Comment (khen27) [inline src/pages/groupView/groupView.ts]: nit: when making comments in code, it's best to stay away from personal pronouns, I would generalize this example if you want it to be pushed to the repo.  Personal pronouns are fine when you're working on your local repository.

File: src/pages/groupView/groupView.ts [hunk 7]
```diff
@@ -8,8 +8,9 @@ import { GroupChatView } from '../groupChatView/groupChatView';
...
 	groups : Array<Object>;
+	//I'm doing this for the purpose of having navigation to the groupChatView page.
 	groupChatView = GroupChatView;
```
Comment (khen27) [inline src/pages/groupView/groupView.ts]: nit: trailing white space

File: src/pages/groupView/groupView.html [hunk 8]
```diff
@@ -1,19 +1,11 @@
 <ion-header>
```
Comment (grant-braqs) [inline src/pages/groupView/groupView.html]: Make sure to add the proper header to your files. Refer to the BRAQS Coding Standards.

File: src/pages/groupView/groupView.scss (hunk 5 above)
Comment (grant-braqs) [inline src/pages/groupView/groupView.scss]: Looking at your css file, I don't think you need comments. 
@evanBRAQS Thoughts here?

File: src/pages/groupView/groupView.ts (hunk 6 above)
Comment (grant-braqs) [inline src/pages/groupView/groupView.ts]: Similar to a resume, you could approach it by starting sentences with a verb. 
"Defining groups using an array of objects."

**Pull request merged**